specification/security/resource-manager/Microsoft.Security/preview/2024-07-01-preview/securityConnectors.json

```diff
@@ -0,0 +1,1707 @@
+              "description": "Is Vulnerability Assessment auto provisioning enabled"
+            },
+            "configuration": {
+              "type": "object",
+              "description": "configuration for Vulnerability Assessment autoprovisioning",
+              "properties": {
+                "type": {
+                  "type": "string",
+                  "description": "The Vulnerability Assessment solution to be provisioned. Can be either 'TVM' or 'Qualys'",
+                  "enum": [
+                    "Qualys",
+                    "TVM"
+                  ],
+                  "x-ms-enum": {
+                    "name": "type",
+                    "modelAsString": true
+                  }
+                }
+              }
+            }
+          }
+        },
+        "mdeAutoProvisioning": {
+          "type": "object",
+          "description": "The Microsoft Defender for Endpoint autoprovisioning configuration",
+          "properties": {
+            "enabled": {
+              "type": "boolean",
+              "description": "Is Microsoft Defender for Endpoint auto provisioning enabled"
+            },
+            "configuration": {
+              "type": "object",
+              "description": "configuration for Microsoft Defender for Endpoint autoprovisioning",
+              "properties": {}
+            }
+          }
+        },
+        "subPlan": {
+          "type": "object",
+          "description": "configuration for the servers offering subPlan",
+          "properties": {
+            "type": {
+              "type": "string",
+              "description": "The available sub plans",
+              "enum": [
+                "P1",
+                "P2"
+              ],
+              "x-ms-enum": {
+                "name": "subPlan",
+                "modelAsString": true
+              }
+            }
+          }
+        },
+        "vmScanners": {
+          "type": "object",
+          "description": "The Microsoft Defender for Server VM scanning configuration",
+          "allOf": [
+            {
+              "$ref": "#/definitions/vmScannersAws"
+            }
+          ],
+          "properties": {}
+        }
+      }
+    },
+    "defenderFoDatabasesAwsOffering": {
+      "type": "object",
+      "description": "The Defender for Databases AWS offering",
+      "x-ms-discriminator-value": "DefenderForDatabasesAws",
+      "allOf": [
+        {
+          "$ref": "#/definitions/cloudOffering"
+        }
+      ],
+      "properties": {
+        "arcAutoProvisioning": {
+          "type": "object",
+          "description": "The ARC autoprovisioning configuration",
+          "allOf": [
+            {
+              "$ref": "#/definitions/arcAutoProvisioningAws"
+            }
+          ],
+          "properties": {}
+        },
+        "rds": {
+          "type": "object",
+          "description": "The RDS configuration",
+          "properties": {
+            "enabled": {
```

Comment:
same as other comments for boolean across the page.

Reply:
This version is build on top of the older version with the goal of supporting a new environment only, without the intention to make further unrelated changes